### ISSUE TITLE
A Question about Research mi_lira_2021: why set training=True at Inference

### ISSUE THREAD

**Issue by Shua-Kang** [tensorflow/privacy#427]
Hi, I have a question about the code of Research mi_lira_2021.
It set training=True at inference:
https://github.com/tensorflow/privacy/blob/4dd8d0ffde4ddb1575d5c2fc02e0693e08f4f4a1/research/mi_lira_2021/inference.py#L72

But since the model contains batchnorm layers, it should be training=True to avoid updating the parameters in the inference.
So why training=True here?
